requirements.txt

```diff
@@ -0,0 +1,23 @@
+blindspin==2.0.1
+certifi==2020.6.20
+chardet==3.0.4
+colorama==0.4.3
+crayons==0.3.1
+deprecation==2.1.0
+docker==4.2.2
+grpcio==1.30.0
+grpcio-tools==1.30.0
+idna==2.10
+packaging==20.4
+protobuf==3.12.2
+PyMySQL==0.9.3
+pyparsing==2.4.7
+pypiwin32==223
+pywin32==228
+requests==2.24.0
+six==1.15.0
+testcontainers==3.0.3
+urllib3==1.25.9
+websocket-client==0.57.0
+Werkzeug==1.0.1
+wrapt==1.12.1
```

Comment:
Do you think that we should check in this file? I saw this is Windows-specific, and in my side, it should be Mac-specific, this file keeps changing from developer to developer

Reply:
And it seems to be created automatically in the script, so don't need to check in

Reply:
The point of this file is for the build script to work. If you don't have this file, the other 4 scripts are pointless. It tells them what libraries need to be installed in the virtual env so that the dev doesn't have to go file-to-file to see what they need to install

Reply:
If we don't check it in, a new developer/user who is just downloading the source will not have a way to set up their venv

Reply:
According to your doc, if the developers can `2. Execute the build_requirements script relevant to your OS.` to generate the `requirements.txt`, why should we check the `requirements.txt` in? Other developers can also execute the same script to generate it, right?